master/buildbot/spec/indent.py

```diff
@@ -5,7 +5,9 @@
 import sys
 
 spaces = re.compile("^ +")
-for fn in sys.argv[1:]:
+
+
+def indent_file(fn: str) -> None:
     lines = []
```

Comment:
Moving this into a function makes this a local variable instead of a loop variable.

Reply:
Also everything is now type checked.